CONTRIBUTING.md

```diff
@@ -0,0 +1,113 @@
+# Contributing to terraform-provider-vcd
+
+The **terraform-provider-vcd** project team welcomes contributions from the community. 
+Before you start working with any contribution, please read our [Developer Certificate of Origin](https://cla.vmware.com/dco).
+All contributions to this repository must be signed as described on that page. Your signature certifies that you wrote the patch or have the right to pass it on as an open-source patch.
+
+
```

Comment:
What about adding a short sentence how it is done here (git commit --signoff) and/or a link to "Updating Pull Requests" section for full details?

Reply:
done